cypress/integration/create.js

```diff
@@ -0,0 +1,29 @@
+describe('Sourcebooks login', function() {
+
+    it('Admin should create a new client', function () {
+
+        const organizationName = Math.ceil(Math.random()*100000);
```

Comment:
Making a random name will work 99.99% of time for not getting a duplicate name, even better approach would be something like:
`const taskName = 'task ' + Date.now();`
`Date.now()` gives us a number of milliseconds from January 1, 1970 which is better than a random number (random number could theoretically repeat). This is a minor issue :)